content/en/service_management/workflows/trigger.md

```diff
@@ -55,6 +58,16 @@ To run the workflow:
 1. Under **Execution parameters**, any template variables you mapped to workflow inputs are automatically populated. Enter the values for any unmapped execution parameters, or edit the existing values if needed.
 1. Click **Run** to run the workflow.
 
+## Trigger a workflow from a workflow
+
+You can trigger a child workflow from another workflow using the **Trigger Workflow** action. For example, if you have a workflow that creates a Jira ticket and then sends a Slack message, there's no need to recreate that workflow for all of your monitors. Instead, create the workflow once and trigger it from everywhere else using the Trigger Workflow action.
```

Comment:
Here's a updated version: "You can trigger a child workflow from another workflow using the **Trigger Workflow** action. For example, if you have a complex series of steps that you need to reuse in several workflows, there's no need to recreate those steps for all of your workflows. For each maintenance in the future, just create those steps once and trigger them from all other workflows using the Trigger Workflow action. Note: this does register as a new workflow execution for billing purposes."